homeassistant/components/zwave_js/api.py

```diff
@@ -436,6 +442,76 @@ def filename_is_present_if_logging_to_file(obj: dict) -> dict:
     return obj
 
 
+@websocket_api.require_admin  # type: ignore
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/start_listening_logs",
```

Comment:
Make it clear from the type that it's a subscription:

```suggestion
        vol.Required(TYPE): "zwave_js/subscribe_logs",
```